src/main/resources/archetype-resources/react-app/src/tests/RouteHelper.test.js

```diff
@@ -0,0 +1,156 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
As they are now under `/src`, are the test files correctly ignored from the production build?

Reply:
Ideally a test belongs to the code and IMO it should run in dev and production build.
Am I wrong? What is the reason the tests should be ignored from production build?

Reply:
To avoid having the test sources included and unnecessarily increase the size of the script that will run in production

Reply:
Jest searches for tests by default in:
... cq/aem-spa-project-archetype/target/test-classes/projects/react/project/test-spa-react/react-app/src/**/__tests__/**/*.{js,jsx,ts,tsx},.../cq/aem-spa-project-archetype/target/test-classes/projects/react/project/test-spa-react/react-app/src/**/?(*.)(spec|test).{js,jsx,ts,tsx}
So this would strengthen my opinion that the unit test are productional code. WDYT? @pfauchere?

Reply:
@pfauchere I cannot find any test code in the js file after installing the react module.